.vsts-dotnet.yml

```diff
@@ -147,6 +147,13 @@ stages:
         ArtifactName: logs
       condition: succeededOrFailed()
 
+    - task: PublishBuildArtifacts@1
+      displayName: 'Publish Artifact: symbols'
+      inputs:
+        PathtoPublish: 'artifacts\bin'
+        ArtifactName: symbols
+      condition: succeededOrFailed()
```

Comment:
To differentiate from...unfinished? I'm guessing this variable is set externally, but if we have access to it, is it possible to change it to finished?

Reply:
This is not part of the PR; GitHub has gotten confused about what commits will actually be merged to 16.5 here.

But the answer to your question is "to distinguish from 'cancelled'", where you might not want to bother uploading partial artifacts, even if you want them from completed-but-failed builds. https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#succeededorfailed